commons-db/src/main/java/com/palantir/nexus/db/sql/BasicSQLUtils.java

```diff
@@ -328,7 +329,7 @@ public static void toStringSqlArgs(final StringBuilder sb, Object[] args) {
                         @SuppressWarnings("resource") // Caller is responsible for closing
                         ByteArrayInputStream bais = (ByteArrayInputStream) arg;
                         byte[] bytes = new byte[bais.available()];
-                        bais.read(bytes, 0, bais.available());
+                        ByteStreams.readFully(bais, bytes, 0, bais.available());
```

Comment:
This automated refactor throws an IOException that won't ever actually happen when reading from the `ByteArrayInputStream`

```suggestion
                        try {
                            ByteStreams.readFully(bais, bytes, 0, bais.available());
                        } catch (IOException e) {
                            throw new SafeRuntimeException("Unable to read SQL args from ByteArrayInputStream", e);
                        }
```